front_camera/front_camera/yellow_rectangle_detecter.py

```diff
@@ -0,0 +1,52 @@
+"""
+動画中の背景下部の黄色検出クラス
+@author: bizyutyu
+"""
+
+import cv2
+
+
+
+class YellowRectangleDetecter:
```

Comment:
detecter ->detecorですかね？
```suggestion
class YellowRectangleDetector:
```

Reply:
修正しました、早期発見ありがとう🙏